snakemake/workflow.py

```diff
@@ -243,7 +250,13 @@ def norm_rule_relpath(f, rule):
         for f in self.included:
             f = local_path(f)
             if f:
-                files.add(os.path.relpath(f))
+                if not (
+                    ON_WINDOWS
+                    and os.path.splitdrive(f)[0] != os.path.splitdrive(os.getcwd())[0]
+                ):
+                    # relpath doesn't work on win if files are on different drive
+                    f = os.path.relpath(f)
```

Comment:
What does doesn't work mean in that case? Is it returning a wrong path or failing with an exception? In the latter case, I think it would be more pythonic to catch the exception and add special windows handling for it.

Reply:
True. I will add that.